src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -125,3 +127,209 @@ def compute_reference(frames: np.ndarray,
         )
 
     return ref_image
+
+
+def optimize_motion_parameters(initial_frames: np.ndarray,
+                               smooth_sigmas: np.array,
+                               smooth_sigma_times: np.array,
+                               suite2p_args: dict,
+                               logger: callable = None) -> dict:
+    """Loop over a range of parameters and select the best set from the
+    max acutance of the final, average image.
+
+    Parameters
+    ----------
+    initial_frames : numpy.ndarray, (N, M, K)
+        Smaller subset of frames to create a reference image from.
+    smooth_sigmas : numpy.ndarray, (N,)
```

Comment:
Small nit: what do we think the risk is that a future developer thinks this `N` is the same as the `N` in `initial_frames.shape`?

Reply:
It's left over from me adhering LSST style guide. https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-array I think it will be pretty clear type of array and rest of the documentation what is going on.